library/src/scala/tasty/util/TreeTraverser.scala

```diff
@@ -0,0 +1,26 @@
+package scala.tasty.util
+
```

Comment:
Do we need this, or is a `TreeAccumulator` good enough?

Reply:
I fell like it would be simpler for users. Personally I have used more times the traversers than the accumulators in the compiler.

Reply:
It also works as an example for users to see how to extend the `TreeAccumulator`.